src/App.tsx

```diff
@@ -177,8 +183,10 @@ const App = () => {
       if (!semver.valid(lastSeenReleaseNotesVersion) || semver.lt(lastSeenReleaseNotesVersion, version)) {
         setAppVersion(version);
         setShowReleaseNotes(true);
+        dispatch(handleWalkThroughStep(1));
       }
     });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Nitpick: I dislike disabling eslint, maybe `useEffectOnce` ([docs](https://github.com/streamich/react-use/blob/master/docs/useEffectOnce.md)) is more descriptive of what we want?

Reply:
In this specific case I think yes, the `useEffectOnce` could be used, but there are cases where we disable the `exhaustive-deps` so that the useEffect is triggered only on some of the dependencies.

With that in mind and not related to this PR, I'm wondering if there are alternative ways to solve the other places where we disable this lint rule.

Reply:
It depends a bit case-by-case, sometimes there is even no way around it and Monokle is not your average web application so I expect it here from time to time.